dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx

```diff
@@ -534,7 +534,7 @@ const styles = {
 	) => css`
 		background: ${background};
 		color: ${textColor};
-		${limitHeight ? 'max-height: 70vh;' : 'auto'}
```

Comment:
the 'auto' was invalid css

Reply:
my bad! 🤦‍♀️

Reply:
the auto bit wasn't causing any issues - browsers just ignore it!
as far as I know the 70% rule was still working